cmd/lakectl/cmd/branch_list.go

```diff
@@ -17,9 +17,11 @@ var branchListCmd = &cobra.Command{
 	Run: func(cmd *cobra.Command, args []string) {
 		amount := Must(cmd.Flags().GetInt("amount"))
 		after := Must(cmd.Flags().GetString("after"))
+		prefix := Must(cmd.Flags().GetString("prefix"))
```

Comment:
If Prefix appears first in the params list, better to keep the same order and place it before `after` (pun intended) here as well.

I know that `amount` doesn't keep the same order - 
Worth also sorting this one.